src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -35,10 +35,11 @@
     private static final int DEFAULT_HEALTH_SERVER_PORT = 8080;
 
     public static void main(String[] args) {
+        log.info("Apache Kafka Bridge {} is starting", Application.class.getPackage().getImplementationVersion());
```

Comment:
I'm quite sure we should not call this `Apache Kafka Bridge`. At least `Strimzi Kafka Bridge` would be needed.

Reply:
but in the CO with don't have Strimzi Cluster Operator, we have just Cluster Operator.
So in this case we can just say Bridge or Kafka Bridge ... is Apache Kafka which is not ok?